lib/message_bus_client_worker/services/polling/generate_params.rb

```diff
@@ -9,7 +9,8 @@ class GenerateParams
       executed do |c|
         c.params = { dlp: 't' }
         c.form_params = c.subscriptions.each_with_object({}) do |sub, hash|
-          hash[sub[0]] = GetLastId.(c.host, sub[0])
+          custom_message_id = sub[1][:message_id] ? sub[1][:message_id].to_s : nil
+          hash[sub[0]] = custom_message_id || GetLastId.(c.host, sub[0])
```

Comment:
This code will always start using the custom id. We want it to continue where it left off if there is something Redis